src/deepsparse/transformers/README.md

```diff
@@ -0,0 +1,268 @@
+Example:
+```bash
+sparseml.transformers.export_onnx --task question-answering --model_path model_path
+```
+This creates `model.onnx` file, in the parent directory of your `model_path`(e.g. `/trained_model/model.onnx`)
+
+####  Directly using the SparseZoo stub
+Alternatively, you can skip the process of onnx model export by downloading all the required model data directly from Neural Magic's [SparseZoo](https://sparsezoo.neuralmagic.com/).
+Example:
+```bash
+from sparsezoo import Zoo
+
+# you can lookup an appropriate model stub here: https://sparsezoo.neuralmagic.com/
+model_stub = "zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_quant_6layers-aggressive_96"
+# directly download the model data to your local directory
+model = Zoo.download_model_from_stub(model_stub)
+
+# the onnx model file is there, ready for deployment
+import os 
+os.path.isfile(os.path.join(model.dir_path, "model.onnx"))
+>>True
+```
+
+
+## Deployment
+
+### Python API
+Python API is the default interface for running the inference with the DeepSparse Engine.
+
+The SparseML installation provides a CLI for sparsifying your models for a specific task; appending the `--help` argument displays a full list of options for training in SparseML:
+```bash
+sparseml.transformers.token_classification --help
+```
+Output:
+```bash
+  --model_name_or_path MODEL_NAME_OR_PATH
+                        Path to pretrained model, sparsezoo stub. or model identifier from huggingface.co/models (default: None)
+  --distill_teacher DISTILL_TEACHER
+                        Teacher model which needs to be a trained NER model (default: None)
+  --cache_dir CACHE_DIR
+                        Where to store the pretrained data from huggingface.co (default: None)
+  --recipe RECIPE       
+                        Path to a SparseML sparsification recipe, see https://github.com/neuralmagic/sparseml for more information (default: None)
+  --dataset_name DATASET_NAME
+                        The name of the dataset to use (via the datasets library) (default: None)
+  ...
+```
+As indicated above, `model_path` may be a path to a local model directory, however, in the examples below, we set the `model_path` argument to the model stub of our SparseZoo models. 
+
+
+#### Question Answering Pipeline
+
+[List of the Hugging Face SparseZoo Question Answering Models](
+https://sparsezoo.neuralmagic.com/?page=1&domain=nlp&sub_domain=question_answering)
+
+```python
+from deepsparse.transformers import pipeline
+
+model_path="zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/12layer_pruned80_quant-none-vnni"
+
+qa_pipeline = pipeline(
+    task="question-answering",
+    model_path=model_path)
+
+inference = qa_pipeline(question="What's my name?", context="My name is Snorlax")
+
+>> {'score': 0.9947717785835266, 'start': 11, 'end': 18, 'answer': 'Snorlax'}
+```
+
+#### Text Classification Pipeline
```

Comment:
Let's call out sentiment analysis as a separate pipeline and use a text classification example for mnli or qqp